### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by richard-thonig**
I talked to Johan. He suggested to add some more detail on the landing page, to explain how to navigate the site a bit better. 
Here we should also explain what a .json file is and how to open .md files (another problem that Johan had, we rely on them in the dowload; worst case we need to add a txt version?)
 what is a good cross-platform way to do this? requiering useres to downloading atom is probably a bit of an overkill.

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ layout: home
 
 <img style="display: block; margin: 0 auto" width="200" src="./images/logo.png" alt="CSP.guru Logo" align="center">
 
-{% assign description = site.data.datapackage["description"] %}
+{% assign description = site.data.datapackage["long_description"] %}
```
Comment (timtroendle) [inline index.md]: I wouldn't hide the download button underneath a lengthy description. I suggest to keep this short description and the download button, and add the long description underneath the download button.

Comment (timtroendle) [inline index.md]: Also, if we add the long description here, I'd suggest to remove the `About` section as it repeats the exact same thing.

Comment (richard-thonig) [reply index.md]: sure thing. in this sense we move: about to the main page.

Comment (richard-thonig) [reply index.md]: seems reasonable.

**Comment by timtroendle** [comment]
> Here we should also explain what a .json file is and how to open .md files (another problem that Johan had, we rely on them in the dowload; worst case we need to add a txt version?)
what is a good cross-platform way to do this? requiering useres to downloading atom is probably a bit of an overkill.

Those are several changes for which I suggest to create separate pull requests and/or issues. Briefly my opinion: 

* All text editors can open `.md`, even the build in text editors on Mac and Windows. Nevertheless I wouldn't mind switching to `.txt` -- it's only file suffix nothing more.
* Regarding the .json file, I wouldn't explain all details here to not repeat ourselves. I think we should mainly do three things: 
** say that we are following the datapackage standard
** link to the datapackage standard for more information
** say that the dataset is the csv file, and its metadata is in the json

**Comment by richard-thonig** [comment]
If I'm closing without merging, this should be burried, right?

**Comment by timtroendle** [comment]
> If I'm closing without merging, this should be burried, right?

Right. But instead of closing you could have amended this pull request. Either way, both will work.

**Pull request closed without merging**